DetectorDescription/Core/interface/DDDivision.h

```diff
@@ -91,45 +92,35 @@ class DDDivision : public DDBase<DDName, DDI::Division*>
    */
   DDDivision(const DDName & name,
              const DDLogicalPart & parent,
-	     const DDAxes axis,
-	     const int nReplicas,
-	     const double width,
-	     const double offset );
-
+	     DDAxes axis,
```

Comment:
Ok, clang decided that passing DDAxes by value here is ok since it would end up making a copy anyway:
```
https://cmssdt.cern.ch/lxr/source/DetectorDescription/Core/src/Division.h#0044
```